### PR TITLE
NO-TICK :: NCTL :: Small fix easing customization of initial stake

### PR DESCRIPTION
**Pull request by AlexanderLimonov**
Improvements to your workflow after this fix: 
 - change the relevant array in nctl/sh/utils/constants.sh instead of parsing and replacing accounts.toml files

Tested by standing up the network with various settings and observing accounts.toml, as well as running the network to confirm that nothing unexpected happened.

### CODE REVIEW

File: resources/production/config-example.toml [hunk 1]
```diff
@@ -65,7 +65,7 @@ bind_address = '0.0.0.0:35000'
 #
 # Multiple addresses can be given and the node will attempt to connect to each, requiring at least
 # one connection.
-known_addresses = ['18.144.176.168:35000', '13.57.200.251:35000']
+known_addresses = ['54.215.92.118:35000', '52.53.239.81:35000', '18.144.25.45:35000', '13.57.247.23:35000']
```
Comment (goral09) [inline resources/production/config-example.toml:68]: Not part of this PR.

File: resources/production/chainspec.toml [hunk 2]
```diff
@@ -48,7 +48,7 @@ round_seigniorage_rate = [185_334_351, 4_503_599_627_370_496]
 # It is the fraction of validators that would need to equivocate to make two honest nodes see two conflicting blocks as
 # finalized: A higher value F makes it safer to rely on finalized blocks.  It also makes it more difficult to finalize
 # blocks, however, and requires strictly more than (F + 1)/2 validators to be working correctly.
-finality_threshold_fraction = [1, 10]
+finality_threshold_fraction = [1, 3]
```
Comment (goral09) [inline resources/production/chainspec.toml:51]: Not part of this PR.

File: resources/production/chainspec.toml [hunk 3]
```diff
@@ -12,12 +12,12 @@ era_id = 0
...
 # contributing to the seeding of the pseudo-random number generator used in contract-runtime for computing genesis
 # post-state hash.
-name = 'casper-delta'
+name = 'mainnet-release-test-1'
 # Timestamp for the genesis block.  This is the beginning of era 0. By this time, a sufficient majority (> 50% + F/2 —
 # see finality_threshold_percent below) of validator nodes must be up and running to start the blockchain.  This
 # timestamp is also used in seeding the pseudo-random number generator used in contract-runtime for computing genesis
 # post-state hash.
-timestamp = '2020-12-29T20:00:00Z'
+timestamp = '2021-03-02T01:00:00Z'
```
Comment (goral09) [inline resources/production/chainspec.toml:20]: Not part of this PR.

File: resources/production/chainspec.toml [hunk 4]
```diff
@@ -12,12 +12,12 @@ era_id = 0
 # Human readable name for convenience; the genesis_hash is the true identifier.  The name influences the genesis hash by
 # contributing to the seeding of the pseudo-random number generator used in contract-runtime for computing genesis
 # post-state hash.
-name = 'casper-delta'
+name = 'mainnet-release-test-1'
```
Comment (goral09) [inline resources/production/chainspec.toml:15]: Not part of this PR.

File: resources/production/accounts.toml [hunk 5]
```diff
@@ -1,31 +1,42 @@
...
+balance = "1000000000000"
+
+[accounts.validator]
+bonded_amount = "9999000000000000"
+
+[[accounts]]
+# bitgo-test
+public_key = "018267d68f8d249b1430551ecc7b4c176d66f2ba2bf98d5547e7c3accc99375e53"
+balance = "10000000000000000"
+
```
Comment (goral09) [inline resources/production/accounts.toml]: I suppose none of it is part of this PR.

File: node/src/reactor/validator.rs [hunk 6]
```diff
@@ -755,10 +755,16 @@ impl reactor::Reactor for Reactor {
...
+                        Tag::BlockByHeight => {
+                            warn!(%sender, "received get block by height response");
+                            return Effects::new();
+                        }
                         Tag::GossipedAddress => {
-                            warn!("received get request for gossiped-address from {}", sender);
+                            warn!(%sender, "received get request for gossiped-address");
                             return Effects::new();
                         }
                     },
```
Comment (goral09) [inline node/src/reactor/validator.rs:770]: Also not part of thi PR.

File: Makefile [hunk 7]
```diff
@@ -127,7 +127,7 @@ lint:
 
 .PHONY: audit
 audit:
-	$(CARGO) audit --ignore RUSTSEC-2020-0123
+	$(CARGO) audit --ignore RUSTSEC-2020-0123 --ignore RUSTSEC-2020-0146
```
Comment (goral09) [inline Makefile:130]: If you merge with `dev` you shouldn't have to do this.

**Comment by goral09** [comment]
@AlexanderLimonov Can you explain (in the PR description) what this PR is achieving functionally? My understanding of that constant and its purpose was different.

File: Makefile (hunk 7 above)
Comment (momipsl) [reply Makefile:130]: out of scope for this PR

File: node/src/reactor/validator.rs (hunk 6 above)
Comment (momipsl) [reply node/src/reactor/validator.rs:770]: out of scope for this PR

File: resources/production/chainspec.toml (hunk 4 above)
Comment (momipsl) [reply resources/production/chainspec.toml:15]: out of scope for this PR

File: resources/production/chainspec.toml (hunk 2 above)
Comment (momipsl) [reply resources/production/chainspec.toml:51]: out of scope for this PR

File: resources/production/config-example.toml (hunk 1 above)
Comment (momipsl) [reply resources/production/config-example.toml:68]: out of scope for this PR

**Comment by AlexanderLimonov** [comment]
This constant sets the stakes in generated accounts.toml. Changing this to an array makes it possible to customize it in a single line without changing any of the logic otherwise.

**Comment by AlexanderLimonov** [comment]
Addressed in a different PR
